DataFormats/EgammaCandidates/interface/AODHIPhoton.h

```diff
@@ -0,0 +1,72 @@
+#ifndef AODHIPhoton_h
+#define AODHIPhoton_h
+
+#include "DataFormats/EgammaCandidates/interface/Photon.h"
+
+namespace aod{
+
+  class AODHIPhoton : public reco::Photon {
+
+  public:
+
+  AODHIPhoton() : reco::Photon() {}
+    virtual ~AODHIPhoton() {}
+
+  AODHIPhoton(const reco::Photon& p) : reco::Photon(p) {}
+
+    float cc1() {return cc1_;}
```

Comment:
- all methods should have comments, preferably with /// or //! to be parsed by doxygen
- all methods and variables should be more intuitive: what you have is  isolation sums based on cluster inputs (c), rechit inputs (r), track inputs (t); the cone sizes are encoded in the last digit and could be more clearly said as R01,2 ..
- all methods that don't modify the object should be declared const (``` float methodName(arguments) const { implementation ```)